### PR TITLE
update 2M link

### PR DESCRIPTION
**Pull request by thelight37**
2M HD live stream

### CODE REVIEW

**Review by spongebob849 — APPROVED** [review]

**Review by Central302 — APPROVED** [review]

File: channels/ma.m3u [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="2MMonde.ma" tvg-country="MA" tvg-language="Arabic;French" tvg-logo="https://i.imgur.com/CwwRn4b.png" group-title="General",2M Monde (360p) [Geo-blocked]
-https://cdnamd-hls-globecast.akamaized.net/live/ramdisk/2m_monde/hls_video_ts_hy217612tge1f21j83/2m_monde.m3u8
```
Comment (Dum4G) [inline channels/ma.m3u:3]: We don't remove links if those keep working. This one does work for me

File: channels/ma.m3u [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="2MMonde.ma" tvg-country="MA" tvg-language="Arabic;French" tvg-logo="https://i.imgur.com/CwwRn4b.png" group-title="General",2M Monde (360p) [Geo-blocked]
```
Comment (Dum4G) [inline channels/ma.m3u:2]:
```suggestion
#EXTINF:-1 tvg-id="2MMonde.ma" tvg-country="MA" tvg-language="Arabic;French" tvg-logo="https://i.imgur.com/CwwRn4b.png" group-title="General",2M Monde (360p) [Geo-blocked]
```
```suggestion
#EXTINF:-1 tvg-id="2MMonde.ma" tvg-country="MA" tvg-language="Arabic;French" tvg-logo="https://i.imgur.com/CwwRn4b.png" group-title="General",2M Monde (360p)
```

**Comment by LaneSh4d0w** [comment]
It basically goes back to the Globecast stream.

**Comment by freearhey** [comment]
Closing due to lack of action from the author of the request.

**Pull request closed without merging**